gobblin-api/src/main/java/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -744,6 +745,8 @@
    * Hive registration properties
    */
   public static final String HIVE_REGISTRATION_POLICY = "hive.registration.policy";
+  // Set to true if hive registration is done in taskStateTracker on task commit.
+  public static final String PIPELINED_HIVE_REGISTRATION_ENABLE = "hive.registration.pipelined.enabled";
```

Comment:
I think this can be PIPELINED_HIVE_REGISTRATION and "hive.registration.pipelined" or if you prefer the current name then change ENABLE to ENABLED.

After looking at the rest of the code, it looks like this parameter is not required. When a handler is configured then it should be executed.